ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -12,8 +12,8 @@ ethernet_interfaces:
     l2_mru: 8000
     ip_address: 172.31.255.1/31
     link_tracking_groups:
-      - name: EVPN_MH_ES1
-        direction: upstream
+      direction: upstream
+      names: [EVPN_MH_ES1, EVPN_MH_ES2]
```

Comment:
This would be a breaking change which cannot get merged until 6.0.0. Instead I think we could do:

```yaml
    link_tracking:
      direction: upstream
      groups: [EVPN_MH_ES1, EVPN_MH_ES2]
```

Reply:
Added!